json-lint/schemas/inventory/adservers/contentad.json

```diff
@@ -22,8 +22,8 @@
                 "id": "$1",
                 "d": "$2",
                 "wid": "$3",
-                "width": "$4",
-                "height": "$5"
+                "width": "${4:required for AMP. Ignored in TOUCH}",
+                "height": "${5:required for AMP. Ignored in TOUCH}"
```

Comment:
You can add this also in the `description` field of the properties

Reply:
changed below